helpdesk_timesheet/__manifest__.py

```diff
@@ -0,0 +1,17 @@
+{
+    "name": "Helpdesk Timesheet",
+    "summary": """
+        Add Timesheets to the Helpdesk""",
+    "category": "Helpdesk",
+    "version": "11.0.1.0.0",
+    "depends": ["helpdesk_mgmt", "hr_timesheet"],
```

Comment:
One dependency per line better?

Reply:
i use pyblack fot formatting. that's the reason why:)

Reply:
https://github.com/python/black The new automated benevolent dictator... :-)